saml-core/src/main/java/org/keycloak/saml/processing/core/util/XMLEncryptionUtil.java

```diff
@@ -514,7 +514,7 @@ private static String getXMLEncryptionURLForKeyUnwrap(String publicKeyAlgo, int
             }
         }
         if (publicKeyAlgo.contains("RSA"))
-            return XMLCipher.RSA_v1dot5;
+            return XMLCipher.RSA_OAEP;
```

Comment:
Would this be backwards compatible with clients already setup and working with RSAES-PKCS1-v1_5?